sentences/de/_common.yaml

```diff
@@ -36,8 +36,8 @@ lists:
   temperature:
     range:
       type: "temperature"
-      from: 0
-      to: 100
+      from: -30
```

Comment:
@synesthesiam how do you prefer to handle this? If we want to avoid matching nonsense values, we could consider having different temperature ranges that we expect to be called for Climate/Oven/Fridge.

Reply:
(we can adjust this later btw, this comment doesn't block this PR)

Reply:
We should have different temperature ranges. I'm less concerned about matching, and more about when we generate sentences that don't make sense.